IOMC/EventVertexGenerators/src/BetafuncEvtVtxGenerator.cc

```diff
@@ -140,3 +137,17 @@ void BetafuncEvtVtxGenerator::sigmaZ(double s) {
 }
 
 TMatrixD const* BetafuncEvtVtxGenerator::GetInvLorentzBoost() const { return &boost_; }
+
+void BetafuncEvtVtxGenerator::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  desc.add<double>("X0Incm", 0.0);
+  desc.add<double>("Y0Incm", 0.0);
+  desc.add<double>("Z0Incm", 0.0);
+  desc.add<double>("SigmaZIncm", 0.0);
+  desc.add<double>("BetaStarIncm", 0.0);
+  desc.add<double>("EmittanceIncm", 0.0);
+  desc.add<double>("TimeOffsetInns", 0.0);
```

Comment:
tests seem to mismatch with the expectations:

https://github.com/cms-sw/cmssw/blob/1a98ef1546db8d226757c780ce3fa6b70e8f9204/IOMC/EventVertexGenerators/src/BetafuncEvtVtxGenerator.cc#L32-L38

Reply:
Thanks! I'll fix the `fillDescription` immediately.

Reply:
while you are at it, you might use `->setComment("in cm")` ;)